5-Heroku/server/models/todoDAO.js

```diff
@@ -0,0 +1,77 @@
+const mongoose = require('mongoose')
+const todoScheme = new mongoose.Schema({
+    todoText: {
+        type: String,
+        required: true
+    },
+    todoId: {
+        type: String,
+        required: true
+    },
+    checked: {
+        type: Boolean,
+        required: true
+    },
+    userId: {
+        type: String,
+        required: true
+    }
+})
+
+const Todo = mongoose.model('Todo', todoScheme)
+
+const modelGetTodoById = async (userId, todoId) => {
+    try {
+        const todo = await Todo.where({userId: userId}).findOne({'todoId': todoId}).lean(true);
+        const resTodo = {
+            todoId: todo.todoId,
+            todoText: todo.todoText,
+            checked: todo.checked
+        }
+        return resTodo;
+    } catch (err) {
+        return { error: "Todo is not found!" }
```

Comment:
throw an exception not a regular object
`throw new Error("Todo is not found", err)`
do the same for all other catch blocks